.travis.yml

```diff
@@ -10,9 +10,9 @@ matrix:
   - php: 5.6
     env: WP_TRAVISCI=travis:phpunit WP_VERSION=nightly
   - php: 5.6
-    env: WP_TRAVISCI=travis:phpunit WP_VERSION=latest
```

Comment:
We'll still need the tests to work with `latest` when this is merged to core for `wp-api`?